TeamSnapSDK/SDK/DataTypes/TSL/TSDKTslChat.m

```diff
@@ -41,4 +41,31 @@ +(void)actionSendPushFirebaseid:(NSString *_Nonnull)firebaseId deviceToken:(NSSt
     }
 }
 
++(void)actionLogDeletedFirebaseId:(NSString *_Nonnull)firebaseId deviceToken:(NSString *_Nullable)deviceToken teamId:(NSString *_Nonnull)teamId eventId:(NSString *_Nullable)eventId completion:(TSDKSimpleCompletionBlock _Nullable)completion {
+    
+    //This is a non-standard object.. We never retrieve these from the server.  So normal SDK Actions aren't relevant. This should probably be handled bt TSDK Root links, but I am including it here for organization and discoverability.
+    [[TSDKTeamSnap sharedInstance] rootLinksWithConfiguration:nil completion:^(TSDKRootLinks *rootLinks, NSError * _Nullable error) {
+        if (rootLinks && (error==nil) && (rootLinks.linkTslChats != nil)) {
+            NSURL *tslChatURL = [rootLinks.linkTslChats URLByAppendingPathComponent:firebaseId];
+            NSMutableDictionary <NSString *, NSString *> *data = [NSMutableDictionary dictionaryWithCapacity:5];
+            data[@"team_id"] = teamId;
+            if (eventId != nil) {
+                data[@"event_id"] = eventId;
+            }
+            if (deviceToken) {
+                data[@"device_token"] = deviceToken;
+            }
+            [TSDKDataRequest requestJSONObjectsForPath:tslChatURL sendDataDictionary:data method:@"DELETE" configuration:nil withCompletion:^(BOOL success, BOOL complete, id  _Nullable objects, NSError * _Nullable error) {
```

Comment:
Seems that if we're sending this with the `DELETE` action, this is more than just a 'log'. Is the server relying on us passing this info for some critical firebase action, or is this merely a convenience?

Reply:
The server isn't actually doing a delete. it's not removing any data. it's just logging that a delete was performed so we can figure out if Firebase is losing messages, or if users are deleting them. The SDK call is correctly named. the API call is not optimal (but out of our control).